### PR TITLE
Adds original file metadata to image view

### PR DESCRIPTION
**Pull request by kenoir**
This is useful for commercial folks whose workflow is mainly downloading
images from the grid for 3rd parties.

screenshot from 2016-04-13 12 11 06

### CODE REVIEW

File: kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js [hunk 1]
```diff
@@ -41,6 +41,37 @@ module.controller('grImageMetadataCtrl', [
 
         // Alias for convenience in view
         ctrl.metadata = ctrl.image.data.metadata;
+
+        function formatBytes(bytes) {
+            if(bytes == 0) return '0 Byte';
+
+            const k = 1000;
+            const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```
Comment (akash1810) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js]: :smile: are `TB`+ needed?? (or even possible)

**Comment by akash1810** [comment]
Filename and dimensions are already on the page. Now that we have PNG (or soon) support, `mime type` is great!

I'd be tempted to only have this metadata in a single place, rather than duplicate it...

**Comment by akash1810** [comment]
Also, having dimensions on the RHS may feel a bit odd when you select a crop as the dimensions are for the master image, rather than the crop.

**Comment by kenoir** [comment]
Closing as we've already got this (albeit in an easily missed place).

File: kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js (hunk 1 above)
Comment (kenoir) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js]: Future proof.

**Comment by paperboyo** [comment]
> `mime type` is great

Yep! But let's do some work on reconciling that, colour models of JPGs and PNGs, info about transparency, ICC colour profile into something more human readable than `mime type: image/jpeg`. I'm eager to help. Oh, and let's serve :fries: with it ;-)

**Pull request closed without merging**